### PR TITLE
Add Image to a list of images

### PR DESCRIPTION
**Pull request by supiash1**

### CODE REVIEW

**Comment by dmgarland** [comment]
Was wondering how long it would be before we got invaded by pokemon. Fix the conflicts and ill merge this

**Comment by dmgarland** [comment]
There are a ton of pokemon images that have been committed to the repo. You'll need to do a `git rm -rf public/uploads` and add that directory to your .gitignore file, so that you don't clog up the repo for everyone else.

**Comment by supiash1** [comment]
I have removed the picture location folder and added the entry to git ignore

File: features/step_definitions/edit_profile_steps.rb [hunk 1]
```diff
@@ -42,6 +42,10 @@
   click_button button
 end
 
+When(/^They upload the image$/) do
+  attach_file('profile[avatars][]', 'public/test_image/pokemon.jpg')
+end
+
 Then(/^They see the message "([^"]*)"$/) do |message|
   page.should have_selector ".alert", text: message
```
Comment (dmgarland) [inline features/step_definitions/edit_profile_steps.rb:54]: This is fine but I don't think it actually verifies either that images were saved or are displayed in the profile. I've found some issues with strong parameters, I'll look into it and see if I can figure out what's wrong

**Pull request merged**